merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/GetSimulationEventsAction.java

```diff
@@ -29,7 +31,8 @@
           e.transaction_index,
           e.causal_time,
           e.topic_index,
-          e.value
+          e.value,
+          e.span_index
```

Comment:
One lingering span_index that should be span_id

```suggestion
          e.span_id
```

Reply:
oops, one got by my eyes. Thanks for catching this.